modules/ROOT/pages/clustering/setup/discovery.adoc

```diff
@@ -242,126 +241,72 @@ dbms.cluster.discovery.resolver_type=LIST
+If the procedure fails, read the error message and resolve the issue manually.
+One possible reason for failure is that some members might have had inconsistent `dbms.cluster.discovery.version` settings initially.
 +
-If they are not, wait and try again.
-+
-. Restart server01 again with the new setting `dbms.cluster.discovery.version=V2_OVER_V1`.
-+
-image:discovery-service/in-place-1-v2-over-v1.png[]
-+
-. Run `SHOW SERVERS` and ensure that all three members are alive.
-. Then repeat steps 6 and 7 for servers 2 and 3.
-Ensure that they are set to `dbms.cluster.discovery.version=V2_OVER_V1`.
-Restart them sequentially, not in parallel.
-+
-image:discovery-service/in-place-23-v2-over-v1.png[]
-
-. Similar to step 5, verify that `stateComparison` shows `Matching`.
-
-. Repeat steps 6, 7, 8, 9, restarting servers 1, 2, and 3 sequentially, with the new setting `dbms.cluster.discovery.version=V2_ONLY`
-image:discovery-service/in-place-123-v2-only.png[]
-. Verify that `CALL dbms.cluster.showParallelDiscoveryState()` now shows `V2_ONLY` running.
-Note that `stateComparison` is `N/A` because you do not have v1 to compare states anymore.
-
-[[discovery-v1-to-v2-new-server]]
-=== New server rolling
-The new server rolling requires three running servers and three new servers.
-
-. Start up the three new servers with the setting `dbms.cluster.discovery.version=V1_OVER_V2`.
-+
-image:discovery-service/v1_over_v2.png[]
-+
-The new servers should have settings which are updated as detailed in the xref:clustering/setup/discovery.adoc#_preparation[Preparation] section.
-The discovery addresses should include addresses of the new members, and the previous members.
-+
-As an example, for those using the list resolver, the settings for the new servers should include:
-+
-[source, parameters]
+[source,cypher]
 ----
-dbms.cluster.discovery.resolver_type=LIST
-dbms.cluster.discovery.version=V1_OVER_V2
-
-dbms.cluster.discovery.endpoints=server01.example.com:5000,server02.example.com:5000,server03.example.com:5000,server04.example.com:5000,server05.example.com:5000,server06.example.com:5000
-dbms.cluster.discovery.v2.endpoints=server01.example.com:6000,server02.example.com:6000,server03.example.com:6000,server04.example.com:6000,server05.example.com:6000,server06.example.com:6000
+CALL internal.dbms.cluster.moveToNextDiscoveryVersion();
 ----
-+
-. Using `bolt://`, connect to the system database of servers 4, 5, 6, and run the following procedure.
-This can be done using via `./cypher-shell -a bolt://localhost:7685 -d system` for example.
+. Check the state again:
 +
 [source,cypher]
 ----
 CALL dbms.cluster.showParallelDiscoveryState();
 ----
 +
-The expected result should display `v2ServerCount` as 3.
-`stateComparison` is not expected to match at this stage because the original servers are not visible to the V2 discovery service.
+Now the returned mode for this server must be `V1_OVER_V2` and the `stateComparison` must show that the states are matching.
```

Comment:
once again - 'for this server'. What's about all other servers?

Reply:
https://github.com/neo4j/docs-operations/pull/1861#discussion_r1809060567